projects/media-viewer/src/lib/store/reducers/annotatons.reducer.ts

```diff
@@ -168,6 +170,38 @@ export function reducer (
         commentSearchQueries
       };
     }
+
+    case fromAnnotations.APPLY_COMMENT_SUMMARY_FILTER: {
+      const payload = action.payload;
+      const hasTagFilter = () => {
+        let isFiltered = false;
+        if (payload.tagFilters) {
+          Object.keys(payload.tagFilters).map(filter => {
+            if (payload.tagFilters[filter] && !isFiltered) {
```

Comment:
Could do a filter here for payload.tagFilters[filter] && !isFiltered instead of map

Reply:
don't think so because there are many user tags so u need to loop through each to check if it has been check